tests/message/test_tasks.py

```diff
@@ -197,6 +198,7 @@ def test_task_to_compute_basic(self):
         serialized = shortcuts.dump(ttc, None, None)
```

Comment:
This whole method should be remove. It's already covered by `SerializationMixin`